frontend/src/components/admin/menu/DictionaryManagement.js

```diff
@@ -209,30 +208,36 @@ function DictionaryManagement() {
   const handleUpdateModal = (e) => {
     e.preventDefault();
 
+    if (!componentMounted.current[dictionaryEntry]) {
+      dirtyFieldsRef.current.add("dictEntry");
+    }
+
+    if (!componentMounted.current[isActive]) {
+      dirtyFieldsRef.current.add("isActive");
+    }
+
+    if (!componentMounted.current[localAbbreviation]) {
+      dirtyFieldsRef.current.add("localAbbreviation");
+    }
+
+    const dirtyFields =
+      dirtyFieldsRef.current.size > 0
+        ? `;${[...dirtyFieldsRef.current].join(";")}`
```

Comment:
here we join all the contents of the dirtyFieldsRef using a `;` which is required by the servverside. so the update json to be sent would look something like:

```
{
  "id": "8",
  "selectedDictionaryCategoryId": "3",
  "dictEntry": "gram + rod378",
  "localAbbreviation": "CGV",
  "isActive": "Y",
  "lastupdated": 1717826409272,
  "dirtyFormFields": ";dictEntry;isActive;localAbbreviation"
}
```